.github/workflows/lock.yml

```diff
@@ -0,0 +1,44 @@
+name: 'Lock Inactive Threads'
+
+on:
+  schedule:
+    # midnight UTC, every Wednesday
+    - cron: '0 0 * * 3'
```

Comment:
Running once a week should be plenty. I want to be respectful of GitHub's resources, and minimize the impact on our GitHub Actions quota for this specific repo.